docs/shared/shave.yaml

```diff
@@ -0,0 +1,15 @@
+shave:
+  execution:
+    batchargs:
+      cores: 1
+      walltime: "00:01:00"
+    executable: /path/to/shave_nc.F90
```

Comment:
"Is the executable actually named with a .F90 extension?", he asked, sadly expecting the answer "Yes".

Reply:
Oh interesting - actually, it turns out that they did not do that! There's a CMakeLists in the directory that compiles it to just "shave", let me fix this